CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 ## Upcoming
 
+- Add MessageSendListener
+- Update CustomMessageInputView
+- Update CustomMessageInputView2
```

Comment:
Not the best kind of way to explain what was added; you should explain what was added/improved (briefly) to the library and not include changes to the sample app (nobody cares about that)